test/server/framework/utils.dart

```diff
@@ -0,0 +1,773 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
How much of this file is actually used? (I guess this comment applies to any code copied out of devicelab)

Reply:
Remove unnecessary utils. I left a few that are used by `server.dart`.